openjdk-tomcat-ubuntu-vm/azuredeploy.parameters.json

```diff
@@ -3,16 +3,10 @@
   "contentVersion": "1.0.0.0",
   "parameters": {
     "javaPackageName": {
-      "value": "openjdk-7-jdk"
+      "value": "openjdk-13-jdk"
     },
     "tomcatPackageName": {
-      "value": "tomcat7"
-    },
-    "newStorageAccountName": {
-      "value": "GEN-UNIQUE-8"
-    },
-    "vmDnsName": {
-      "value": "GEN-UNIQUE-8"
+      "value": "tomcat9"
```

Comment:
remove this and use a defaultValue in the template

Reply:
yes completed    

"tomcatPackageName": {
            "type": "string",
            "defaultValue": "tomcat9",
           "allowedValues": [
              "tomcat7",
              "tomcat8",
              "tomcat9"
            ],
            "metadata": {
              "description": "tomcat Package Name"
            }
          },